### DISCUSSION TITLE
Which version of Intervention/image should I use?

### DISCUSSION THREAD

**Discussion by Alex123Sh** [UniSharp/laravel-filemanager#1227] [Q&A]
Good day.
When I try to load a picture, I get a message in the browser (Network) - 

_{message: "Class "Intervention\Image\Facades\Image" not found", exception: "Error",...}
exception: "Error"
file: "C:\\xampp\\htdocs\\laravel_02.loc\\vendor\\unisharp\\laravel-filemanager\\src\\LfmPath.php"
line: 325
message:"Class \"Intervention\\Image\\Facades\\Image\" not found"_

The Facades folder is not in the project at all.
The default version used is Intervention/image 3.5, in version Intervention/image 2.7 there is a Facades folder.

Which version of Intervention/image should I use?
Or how to overcome this error?

![Screenshot_3](https://github.com/UniSharp/laravel-filemanager/assets/5430095/4da7cc18-f8c4-41e9-95bb-1dda1807f423)

**Comment by streamtw** [comment]
Please install `v2.*` versions of `intervention/image` for now. The support for `v3.*` is a work in progress.

**Comment by Alex123Sh** [comment]
YES! Everything worked out, thank you very much!

**Comment by streamtw** [comment]
Update: `v2.9.0` has been released to fully support `v3.*` versions of `intervention/image`.
